operator-metadata/image.yaml

```diff
@@ -3,35 +3,21 @@ schema_version: 1
 name: amq7/amq-streams-rhel7-operator-metadata
 description: "AMQ Streams image containing the operator metadata"
 version: "1.5.0"
-from: scratch
+from: rh-osbs/ubi7-minimal:7-released
 
 labels:
   - name: "com.redhat.component"
-    value: "amqstreams-bundle-container"
+    value: "amqstreams-operator-metadata-container"
   - name: "io.k8s.description"
     value: "Operator metadata"
   - name: "io.k8s.display-name"
     value: "AMQ Streams Operator metadata"
   - name: "io.openshift.tags"
     value: "messaging,amq,jboss"
-  - name: "operators.operatorframework.io.bundle.mediatype.v1"
-    value: "registry+v1"
-  - name: "operators.operatorframework.io.bundle.manifests.v1"
-    value: "manifests/"
-  - name: "operators.operatorframework.io.bundle.metadata.v1"
-    value: "metadata/"
-  - name: "operators.operatorframework.io.bundle.package.v1"
-    value: "amq-streams"
-  - name: "operators.operatorframework.io.bundle.channels.v1"
-    value: "stable"
-  - name: "operators.operatorframework.io.bundle.channel.default.v1"
-    value: "stable"
-  - name: "com.redhat.delivery.operator.bundle"
-    value: "true"
-  - name: "com.redhat.openshift.versions"
-    value: "v4.5"
-  - name: "com.redhat.delivery.backport"
+  - name: "com.redhat.delivery.appregistry"
     value: "true"
+  - name: "com.redhat.delivery.operator.bundle"
```

Comment:
we can remove this one as well, it's not in the image.yaml of 1.4.0-dev branch.

Reply:
Done!